pkgs/development/interpreters/octave/wrap-octave.nix

```diff
@@ -0,0 +1,18 @@
+{ lib
+, octave
+, makeSetupHook
+, makeWrapper
+}:
+
+with lib;
```

Comment:
Is this even used?

Reply:
No, it wasn't. It was a left-over after reading through and copying a combination of Lua's and Python's `wrap-lang.nix` files. It has been removed.